src/schemas/xspec.rnc

```diff
@@ -37,6 +37,11 @@ description =
 		[ a:defaultValue = "import" ]
 		attribute run-as { "import" | "external" }?,
 		
+		## Set this true to report how long it took XSpec to run scenarios.
+		## Requires Saxon-PE or Saxon-EE.
+		[ a:defaultValue = "no" ]
+		attribute timing { boolean.datatype }?,
```

Comment:
The attribute name `timing` is OK, but could be clearer. Some alternatives you might consider are `enable-timing`, `measure-time`, `mark-time`, `report-time`, `report-elapsed`, and `time-scenarios`.

Reply:
c7222ea22e95ae4d061d759fc6424835e4b70ed6 renames it to `measure-time`.

Some other considerations:
- `timing`: Some people may think it enables Saxon's command line `-t` option or `http://saxon.sf.net/feature/timing` feature flag.
- `timeit`: Some developers may be familiar with this name and its functionality. But it may be too vague for the other people.